.flowconfig

```diff
@@ -1,14 +1,15 @@
 [ignore]
 .*/node_modules/.*[^(package)]\.json$
+<PROJECT_ROOT>/dist/.*
 
 [include]
 ./src/
 
 [libs]
+./node_modules/fusion-core/flow-typed
```

Comment:
Are we using this for anything yet? Or just getting our configs to match?

Reply:
Good question- it's used here for `[Context](https://github.com/fusionjs/fusion-test-utils/pull/38/commits/8ba7a6b51553d4e6b0eb380479e853d063339882#diff-9264294e9bb5e1469869da14a9cea86dR5)` and `FusionApp`.  At least internally, it looks like our plan is to ship `libdef` files like we are will soon be doing in `[fusion-core](https://github.com/fusionjs/fusion-core/blob/di/flow-typed/fusion-core.js)`.